src/library_pthread.js

```diff
@@ -388,7 +397,7 @@ var LibraryPThread = {
       };
 
       worker.onerror = function(e) {
-        err('pthread sent an error! ' + e.filename + ':' + e.lineno + ': ' + e.message);
+        err('pthread sent an error! ' + e.stack ? e.stack : (e.filename + ':' + e.lineno + ': ' + e.message));
```

Comment:
This makes the error output from unit tests/node much, much nicer (i.e. actually tells you what failed)